r/R/compute.R

```diff
@@ -264,6 +287,9 @@ is_in.ArrowDatum <- function(x, table, ...) {
 #' @param x `Array` or `ChunkedArray`
 #' @return A `StructArray` containing "values" (same type as `x`) and "counts"
 #' `Int64`.
+#' @examples
+#' cyl_vals <- Array$create(mtcars$cyl)
+#' value_counts(cyl_vals)
```

Comment:
This is less a concern of the examples, and more the code for the `StructArray` print method (I think): I'm a little surprised that the names `value` and `counts` aren't displayed:

```
> value_counts(Array$create(mtcars$cyl))
StructArray
<struct<values: double, counts: int64>>
-- is_valid: all not null
-- child 0 type: double
  [
    6,
    4,
    8
  ]
-- child 1 type: int64
  [
    7,
    11,
    14
  ]
```

Reply:
Yeah, absolutely. It's possible there's some good reason it's not possible, but we could figure that out on the ticket or when we do that ticket

Reply:
https://issues.apache.org/jira/browse/ARROW-12847